services/fdb/fdb.proto

```diff
@@ -615,3 +615,18 @@ message FDBCLIResponse {
     Log log = 2;
   }
 }
+
+service Server {
+  rpc FDBServer(FDBServerRequest) returns (FDBServerResponse) {}
+}
+
+message FDBServerRequest {
```

Comment:
This is fine now, but consider how it'll look if we ever need to add something else into `sanssh fdb fdbserver`.

I think these are better as `FDBServerVersionRequest` and `FDBServerVersionResponse`, and the command-line invocation could be `sanssh fdb fdbserver version`

The generic `FDBServerRequest` seems equivalent to `exec run`, so perhaps we don't need it at all?

Reply:
My original thought was when new actions under fdbserver need to be implemented, we can add the action inside the FDBServerRequest probably name it FDBServerCommand, and the FDBServerCommand will looks like FDBCLICommand: https://github.com/Snowflake-Labs/sansshell/blob/ec9a0e4af5597f44daba1df5ec688b43aecdfd27/services/fdb/fdb.proto#L499 that includes a lot of subcommands. 

Here the version is defined as a flag to fdbserver, I am also fine to put the version as a subaction for fdbserver, in this case the proto will be like :
```go
message FDBServerRequest {
  FDBServerCommand command;
}

message FDBServerCommand {
  oneof command {
     FDBServerVersion version = 1;
  }
}
``` 

Then the action will look like what you suggested `sanssh fdb fdbserver version`.